schemas/Identifier.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "Identifier",
-  "description": "An identifier for a user or institution.",
+  "description": "An identifier for a user or institution. Between 2-20 characters inclusively. May include dashes (-), but not at the beginning or end and no more than one consecutively.",
```

Comment:
I think there's a good chance what we're calling username now will end up being used as a more opaque account identifier in practice. It could be a plain number, but if we're going to use an opaque unique ID anyway, why not use a UUID? We're using UUIDs all over the place in our protocols. Also, I might want to allow users to create account aliases to hand out to different people and I might not want the external party to know whether they've given their "real" account identifier or not.

I don't feel too strongly about this point but it surprised me at first that we wouldn't support UUIDs since that seems to be our go-to for ids.